sklearn/utils/random.py

```diff
@@ -197,3 +199,58 @@ def random_choice_csc(n_samples, classes, class_probability=None,
     return sp.csc_matrix((data, indices, indptr),
                          (n_samples, len(classes)),
                          dtype=int)
+
+
+class loguniform:
+    """
+    A class supporting log-uniform random variables. It generates values
```

Comment:
Pep257 suggests a one-time line summary on the same line as """. Split the second sentence into a different paragraph